SassHandler.php

```diff
@@ -203,15 +203,14 @@ class SassHandler extends CApplicationComponent
      */
     public function init()
     {
-        $vendorPath = __DIR__ . '/../../';

```

Comment:
Composer puts "vendor" directory to the same location where you run `composer install`.
Usually it should be the root directory, not `protected` directory.

But it doesn't matter where composer puts the directory.
Because the final directory structure is:
- vendor/leafo/scssphp
- vendor/artem-frolov/yii-sass

You don't need to exit the vendor directory, all packages are inside.